imap_processing/common_cdf_attrs.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+
+global_base = {
+    "Project": ["STSP Cluster>Solar Terrestrial Science Programmes, Cluster"],
+    "Source_name": ["IMAP>Interstellar Mapping and Acceleration Probe"],
+    "Discipline": ["Solar Physics>Heliospheric Physics"],
+    "Descriptor": ["IDEX>Interstellar Dust Experiment"],
+    "PI_name": ["Dave McComas"],
```

Comment:
```suggestion
    "PI_name": ["David J. McComas"],
```
I assume we should use full qualified names?

Reply:
Fixed, and also added "Dr." because thats how the one IBEX CDF file I found had it